src/@types/event.ts

```diff
@@ -119,6 +119,7 @@ export const RoomCreateTypeField = "type";
 
 export enum RoomType {
     Space = "m.space",
+    Call = "org.matrix.msc3417.call",
```

Comment:
I'd suggest this be called `UnstableCall` so we can introduce the stable version more easily down the line. Alternatively, `UnstableValue` usage outside of this enum (like we do for event types).